crates/edr_napi/src/provider.rs

```diff
@@ -30,6 +33,22 @@ impl Provider {
         logger_config: LoggerConfig,
         #[napi(ts_arg_type = "(event: SubscriptionEvent) => void")] subscriber_callback: JsFunction,
     ) -> napi::Result<JsObject> {
+        START.call_once(|| {
```

Comment:
Does this approach work? I'm confused based on your previous comment. It currently at least still doesn't disable when the tracing feature is off

Reply:
Yeah this works to initialize the tracing config only once, we just can't store the flame drop guard as static. Based on my reading of the code, previously only the flame guard depended on the `tracing` feature in `edr_napi` and basic tracing was always enabled. Always enabling basic tracing is replicated here, but the flame guard is removed.